README.md

```diff
@@ -10,41 +10,50 @@
 If you already have VS Code and Docker installed, you can click the badge above or [here](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=git@github.com:demisto/content.git) to get started. Clicking these links opens VS Code, installs the `Remote-Containers` extension (if not installed), clones the source code into a container volume, and spins up a development container, configured with all recommended settings.
```

Comment:
What is this README and why is it in your change?

Reply:
This is not intended. I think something to do with VS code. I will remove this.

Reply:
Version control gremlins must have gotten in again.

Reply:
Undid those changes. It was just reformatting spaces.